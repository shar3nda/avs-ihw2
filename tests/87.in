l+_;Eo_>u`|G^E]$6 -"FlU-W8IHno~a0$$c7p$`(%hp%(.)u\Wm[iE}~_d11<{8QeN{1f