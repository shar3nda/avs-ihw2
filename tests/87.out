Lowercase: 678, uppercase: 669